ConsoleApp_Task1/Program.cs

```diff
@@ -82,44 +34,68 @@ public static double ReturnRadiusOrSide()
                     {
                         // this class's error string
                         string LastError = sex.Message;
-                        Console.WriteLine("\nIncorrect Input: value is not parsed correctly");
+                        Console.WriteLine("Incorrect Input: value is not parsed correctly");
                         e++;
                     }
                 }
                 else
                 {
-                    Console.WriteLine("\nIncorrect Input: Empty value is not allowed");
+                    Console.WriteLine("Incorrect Input: Empty value is not allowed");
                     e++;
                 }
             }
-
             if (e == 3)
             {
                 double minNumber = 0.5;
                 double maxNumber = 5.0;
-                double RadiusOrSide = new Random().NextDouble() * (maxNumber - minNumber) + minNumber;
-                Figure OtherCircleOrArea = new Figure(RadiusOrSide) { };
-                Console.WriteLine("\nRandom value is " + Math.Round((OtherCircleOrArea.input),2));
-                return OtherCircleOrArea.input;
-
+                _input = new Random().NextDouble() * (maxNumber - minNumber) + minNumber;
+                Console.WriteLine("\nRandom value is " + Math.Round((_input), 2));
+                return _input;
             }
             return 0;
         }
-
+        public class Program
+        {
+            public static void Main(string[] args)
+            {
+                Figure figure = new Figure();
+
+                // To get the Area of the Circle
+                Console.WriteLine("What is the radius of the circle? Please populate and push Enter key \nNOTE:3 attempts are available otherwise random value will be taken");
+                double Radius = figure.GetValidatedInput();
+                Figure circle = new Circle(Radius);
+                double MainResultCircle = circle.GetFigureArea();
+                Console.WriteLine("\n\rThe area of the circle is " + MainResultCircle);
+
+                // To get the Area of the Square
+                Console.WriteLine("\nWhat is the side of the square? Please populate and push Enter key \nNOTE:3 attempts are available otherwise random value will be taken");
+                double Side = figure.GetValidatedInput();
+                Figure square = new Square(Side);
+                double MainResultSquare = square.GetFigureArea();
+                Console.WriteLine("\n\rThe area of the square is " + MainResultSquare);
+
+                // does square fit into the circle/circle fit into the square?
+                string result = "";
+                if (Radius < (Side / 2))
+                {
+                    result = "\r\nThe Circle fits into the Square";
+                }
+                if (Math.Pow(2 * Math.Pow(Side, 2), 1 / 2) < (2 * Radius))
+                {
+                    result = "\r\nThe Square fits into the Circle";
```

Comment:
for me, the program says that a square with 6 side fits the circle with 4 radius.

![image](https://user-images.githubusercontent.com/9382934/111040614-69087c00-843c-11eb-9af1-30bf0ed3ea40.png)

could you please investigate and fix it?